Sources/Clappr_iOS/Classes/Plugin/Core/Drawer/BottomDrawerPlugin.swift

```diff
@@ -65,13 +65,15 @@ open class BottomDrawerPlugin: DrawerPlugin {
     }
 
     private func addTapGesture() {
-        let gesture = UITapGestureRecognizer(target: self, action: #selector(didTapView))
-        view.addGestureRecognizer(gesture)
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(didTapView))
+        tapGesture.cancelsTouchesInView = false
+        view.addGestureRecognizer(tapGesture)
```

Comment:
CodeClimate is complaining about similarity between this method and the one below.
Do you think creating another method would solve this? Something like:
```swift 
    private func addGesture(_ gesture: UIGestureRecognizer) {
        gesture.cancelsTouchesInView = false
        view.addGestureRecognizer(gesture)
    }
```
and passing the gesture to it, like:

```swift
    addGesture(UITapGestureRecognizer(target: self, action: #selector(didTapView)))
```